manifests/4.4/kibanas.crd.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: kibanas.logging.openshift.io
+spec:
+  subresources:
+    status: {}
+  group: logging.openshift.io
+  names:
+    kind: Kibana
+    listKind: KibanaList
+    plural: kibanas
+    singular: kibana
+  scope: Namespaced
+  version: v1
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          type: object
+          description: Specification of the desired behavior of the Kibana cluster
```

Comment:
`Specification for the desired behavior of Kibana`. Leave it at that since Kibana is not a cluster